pages/view/[id].js

```diff
@@ -3,6 +3,7 @@ import Head from 'next/head'
 import { useRouter } from 'next/router'
 import { GlobalContext } from '../../context/GlobalState'
 import AddNoteButton from '../../components/AddNoteButton'
+import { noteDate } from '../../functions/time'
```

Comment:
Please change it to a React Hook and change the folder name to `hooks`.

Reply:
I have thought react hooks used for internal state and with custom hooks, we can have DRY state logic in components.